deny.toml

```diff
@@ -86,7 +86,6 @@ skip = [
     "cookie",
     "futures",
     "redox_syscall",
```

Comment:
I think you need to add objc2-metal and objc2-quartz-core to deny.toml. Shouldn't the linter be catching that?